source/scripting/Vehicle.hpp

```diff
@@ -728,6 +728,14 @@ namespace GTA
 		{
 			float get();
 		}
+		/// <summary>
+		/// Gets the speed the wheels are spinning at
+		/// </summary>
+		/// <value>The speed in metres per second</value>
```

Comment:
Not exactly important, but American English would be preferred, so we stay consistent =P (meters)